src/core/Akka/Actor/RepointableActorRef.cs

```diff
@@ -238,7 +238,7 @@ public class UnstartedCell : ICell
         private readonly object _lock = new object();
 
        /* Both queues must be accessed via lock */
-        private readonly List<Envelope> _messageQueue = new List<Envelope>();
+        private readonly LinkedList<Envelope> _messageQueue = new LinkedList<Envelope>();
```

Comment:
Switched to using a `LinkedList` to conserve memory once the cell starts and the list is drained / empty.